addons/l10n_it_edi/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
-"access_it_ddt_manager","it_ddt manager","model_l10n_it_ddt","account.group_account_invoice",1,1,1,1
\ No newline at end of file
+"access_it_ddt_manager","it_ddt manager","model_l10n_it_ddt","account.group_account_invoice",1,1,1,1
+"access_l10n_it_document_type","access_l10n_it_document_type","model_l10n_it_document_type","base.group_user",1,1,1,1
```

Comment:
The rule used to be on `account.group_account_invoice` for this model. I just want to make sure if this change is on purpose, and if it makes sense. (I'm not against the change, it's just surprising)
```
l10n_it_edi_ndd.access_l10n_it_document_type,access_l10n_it_document_type,l10n_it_edi_ndd.model_l10n_it_document_type,account.group_account_invoice,1,1,1,1
```

Reply:
Ahhhh just saw, no i just copy pasted what the console gave me :eyes: Will change that